doc/Media.xml

```diff
@@ -1629,6 +1629,7 @@ Change Request 2430, 2479</revremark>
     <section>
       <title>Video source configuration</title>
       <para>A VideoSourceConfiguration contains a reference to a VideoSource and a Bounds structure containing either the whole VideoSource pixel area or a sub-portion of it. The Bounds and VideoSource define the image that is streamed to a client. If a VideoSourceConfiguration is used inside a profile its UseCount parameter is increased to indicate that changing this configuration could affect other users.</para>
+      <para>For any children element of the VideoSourceConfiguration that is not defined here, follow the Media2 specification definitions.</para>
```

Comment:
**Suggestion** 
  - For any parameters under VideoSourceConfiguration that are not defined here, follow the Media2 specification for definitions.